src/main/java/com/refinedmods/refinedstorage/network/ServerProxy.java

```diff
@@ -0,0 +1,102 @@
+package com.refinedmods.refinedstorage.network;
+
+import com.refinedmods.refinedstorage.api.network.grid.IGrid;
+import com.refinedmods.refinedstorage.api.network.grid.handler.IItemGridHandler;
+import com.refinedmods.refinedstorage.apiimpl.network.grid.handler.ItemGridHandler;
+import com.refinedmods.refinedstorage.apiimpl.storage.cache.ItemStorageCache;
+import com.refinedmods.refinedstorage.container.GridContainer;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.ItemStack;
+
+import java.util.UUID;
+
+public class ServerProxy {
+    public static void onInventoryScrollMessage(ServerPlayerEntity player, int slot, boolean shift, boolean up) {
```

Comment:
This impl should go in the grid handler, perhaps also add a way to reuse this logic.